integration-tests/src/test/java/com/datastax/oss/driver/mapper/GetEntityIt.java

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
It is not where I want it yet. It will be flushed out when I return

Reply:
There is a problem with compilation of this branch. 
The `SetEntityIT` fails because there is no class `com.datastax.oss.driver.mapper.model.inventory.InventoryMapperBuilder;`
https://travis-ci.com/datastax/java-driver/jobs/180213823

On the other hand from IntelliJ I got:
`Error:java: Bad service configuration file, or exception thrown while constructing Processor object: javax.annotation.processing.Processor: Provider com.datastax.oss.driver.internal.mapper.processor.MapperProcessor could not be instantiated: java.lang.NoClassDefFoundError: com/datastax/oss/driver/internal/mapper/processor/ProcessorContext`